### PR TITLE
Renaming repos

### PR DESCRIPTION
**Pull request by quarkytale**
Changes:
- Updated references to repos
- renamed buoy_all.yaml

Workflow:
Rename repos on Github -> Merge in https://github.com/osrf/buoy_entrypoint/pull/38 -> Wait for green CI here -> Merge in this PR -> Merge in https://github.com/osrf/buoy_msgs/pull/45

### CODE REVIEW

**Review by mabelzhang — APPROVED** [review]

File: .github/workflows/build-and-test.sh [hunk 1]
```diff
@@ -51,5 +49,5 @@ colcon build --packages-up-to buoy_tests --event-handlers console_direct+
 source $COLCON_WS/install/setup.bash
 
 # Test all buoy packages
-colcon test --packages-select-regex=buoy --packages-skip=buoy_msgs --event-handlers console_direct+ --retest-until-pass 10
+colcon test --packages-select-regex=buoy --packages-skip=mbari_wec_utils --event-handlers console_direct+ --retest-until-pass 10
```
Comment (mabelzhang) [inline .github/workflows/build-and-test.sh:52]: Package name did not change
```suggestion
colcon test --packages-select-regex=buoy --packages-skip=buoy_msgs --event-handlers console_direct+ --retest-until-pass 10
```

**Comment by mabelzhang** [comment]
Try to update topic name watch dog to watchdog (one word) as updated in the PR here https://github.com/osrf/mbari_wec_utils/pull/43/files

File: buoy_gazebo/package.xml [hunk 2]
```diff
@@ -12,8 +12,8 @@
 
   <depend>buoy_description</depend>
   <depend>buoy_interfaces</depend>
-  <depend>buoy_msgs</depend>
   <depend>gz-sim7</depend>
+  <depend>mbari_wec_utils</depend>
```
Comment (mabelzhang) [inline buoy_gazebo/package.xml:16]: Remove as that is the repo name

**Comment by quarkytale** [comment]
Needs https://github.com/osrf/mbari_wec_utils/pull/46

**Review by andermi — APPROVED** [review]

**Pull request merged**